### ISSUE TITLE
MultiPoolMiner

### ISSUE THREAD

**Issue by imrecsoka** [aaronsace/MultiPoolMiner#76]
Hi, i just want to say well done, very nice little tool for us to use. 
I have been running it for maybe half  an hour now here in your neighboring country below, just think of David Attenburough;s documentaries :)
Anyways, i have received a few DLL errors but i just click ok and let the thing do its thing.
When i have questions i will ask, but for now i just wanted to say , i just happened to find this and i am grateful for the work you did , thanks.
James

**Comment by snakethejake81** [comment]
You need these:
CCMiner may need 'MSVCR120.dll' if you don't already have it:
https://www.microsoft.com/en-gb/download/details.aspx?id=40784
CCMiner may need 'VCRUNTIME140.DLL' if you don't already have it:
https://www.microsoft.com/en-us/download/details.aspx?id=48145

I usually install both the x86 and x64 just in case.

**Comment by somidax** [comment]
Hi,
Even when the two is installed, it still give the run time dll errors.

A further look and investigation,I observed most mining software are optimizing it to be standalone which is noticeable with Trupvot ccminer 2.0.

Note that most installed .dll could be reprogrammed to start other software so when using MPM  this pop-up comes up telling you it is missing while actually it is in your system.

I tried working on optimize .dll and Inject into the miner, it works sometime and sometimes it won't after rebooting or starting the mining rig.

I really appreciate his work but while he is working to perfect it, I simply inject my .dll into d app and if it works, I keep mining but when it stops and ask for .dll? I reboot the system.

I know he will surely fix it for we are to encourage him to keep up d good work.
